benchmark_scenarios/eurac_pv_farm_detection.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+      "id": "eurac_pv_farm_detection",
+      "type": "openeo",
+      "description": "ML photovoltaic farm detection, developed by EURAC",
+      "backend": "openeofed.dataspace.copernicus.eu",
+      "process_graph": {
+        "maxndvi1": {
+          "process_id": "eurac_pv_farm_detection",
+          "namespace": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/refs/heads/hv_pv_detection/openeo_udp/eurac_pv_farm_detection/eurac_pv_farm_detection.json",
```

Comment:
Note that this reference will break once the `hv_pv_detection` feature branch is removed

also see https://github.com/ESA-APEx/apex_algorithms/issues/48

Reply:
Yes, so is the idea that once its merged we do a follow-up PR? Or is there an alternative solution?

Reply:
I think, if you use a "permalink", it will keep working after the feature branch is removed

in this case that would be 
> https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/4003046e3b79ec3ab8dace888a231655db389d66/openeo_udp/eurac_pv_farm_detection/udf_eurac_pvfarm_onnx.py

it's a bit cumbersome to explain how to get that from within this comment, but we can discuss this elsewhere